src/tests/unit/tests/popup/components/ad-hoc-tools-panel.test.tsx

```diff
@@ -24,6 +24,7 @@ describe('AdHocToolsPanelTest', () => {
         const props: AdHocToolsPanelProps = {
             backLinkHandler: null,
             diagnosticViewToggleFactory: diagnosticViewToggleFactoryMock.object,
+            featureFlagStoreData: {},
```

Comment:
We should add a new test case that tests what happens when the feature flag is present.